### DISCUSSION TITLE
How can I create a bulk action

### DISCUSSION THREAD

**Discussion by diegoba73** [Power-Components/livewire-powergrid#195] [Ideas]
Hello, how can I do a batch data update action? Thank you

**Comment by luanfreitasdev** [comment]
Hello @diegoba73 . 

For now this is not possible. But if you want to send a PR we would be very grateful.

**Comment by dansysanalyst** [comment]
@diegoba73 there is now an example of Bulk Actions in the [PowerGrid Demo Repository](https://github.com/Power-Components/powergrid-demo).

<img width="976" alt="CleanShot 2022-02-16 at 21 22 19@2x" src="https://user-images.githubusercontent.com/79267265/154353578-9b810bc9-0e1a-429d-a930-b525e5c72057.png">

**Comment by dansysanalyst** [comment]
I have made it available on the doc: https://livewire-powergrid.com/#/table/bulk-actions?id=usage

---

⭐ If you are enjoying PowerGrid consider Starring the repository!
